stories/Checkbox.stories.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+
+import { withDesign } from 'storybook-addon-designs';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { ThemeProvider } from 'styled-components';
+
+import baseTheme from '../app/themes/baseTheme';
+import Checkbox from '../app/components/elements/Checkbox';
+import { CheckboxGroupTitle } from '../app/components/elements/FontStyles';
+
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
I'm hesitant to disable accessibility checks without a solid reason.  I think we really should be doing what we can to enforce a11y practices in our new components, and see our storybook serving as examples of best practices of how to use our components so in instances where we can't enforce it via prop-types, we have examples to copy from that have a11y practices built-in.